app/src/main/res/layout/browser_overlay.xml

```diff
@@ -41,6 +41,18 @@
             android:src="@drawable/turbo_selector"
             android:contentDescription="@string/turbo_mode"/>
 
+        <View
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_weight="1" />
```

Comment:
perf nit: there's an IDE warning for nested LinearLayouts where both use `layout_weight`s:

![screen shot 2018-02-26 at 14 49 03](https://user-images.githubusercontent.com/759372/36700345-3cb7b37c-1b04-11e8-92b9-60d4ed1bc17d.png)

That being said, I feel like layout perf is trivial for self-containing views like this (unless it gets in the way of animations) so it's probably good enough and I wouldn't bother to change it. fwiw, I might design this as a root FrameLayout with four sub-containers for each corner. ConstraintLayout is also an option. I'm not sure how this designs would affect keyboard navigation, however.

Reply:
@mcomella I tried to go down the route of converting it into a ConstraintLayout but it started to increase the scope of this PR a bit. I definitely think we should do it as a follow up item though. What do you think?

Reply:
Tracking #540